streampipes-model/src/main/java/org/apache/streampipes/model/schema/EventProperty.java

```diff
@@ -85,6 +86,48 @@
 
   private String runtimeId;
 
+  private boolean equateLists(List<?> list1, List<?> list2) {
```

Comment:
Why don't we use `list1.equals(list2)`?

Reply:
I have used ListUtils.isEqualList for comparing the lists now(requires extra dependency). Since we are already overriding equals everywhere for this story.
Is this fine?Or you want me to sort the list for which we have to write the comparator function and then do list1.equals(list2).